src/sentry/utils/email.py

```diff
@@ -41,6 +42,11 @@
 # The maximum amount of recipients to display in human format.
 MAX_RECIPIENTS = 5
 
+# The fake TLD used to construct email addresses when one is required,
+# for example by automatically generated SSO accounts.
+FAKE_EMAIL_DOMAIN = 'sentry-fake'
```

Comment:
Being pedantic here, but this isn't a domain. It's just a TLD.

Reply:
@dcramer any thoughts on making this a configurable option? Not sure if there'd be any reason someone may want to override this, or even us for sentry.io. This seems fine to me.